### ISSUE TITLE
Does not parse --platform flag in FROM instruction

### ISSUE THREAD

**Issue by dsherret** [HewlettPackard/dockerfile-parser-rs#18]
I was looking at the Dockerfile reference and [here](https://docs.docker.com/engine/reference/builder/#from) it says `FROM` instructions have the following form:

```
FROM [--platform=<platform>] <image> [AS <name>]
# or
FROM [--platform=<platform>] <image>[:<tag>] [AS <name>]
# or
FROM [--platform=<platform>] <image>[@<digest>] [AS <name>]
```

I don't see this `--platform` flag supported by the code and when I try to parse `FROM --platform=$BUILDPLATFORM image` it errors with:

```
could not parse Dockerfile:  --> 2:16
  |
2 | FROM --platform=$BUILDPLATFORM image␊
  |                ^---
  |
  = expected EOI
```

I don't know much about dockerfiles though, so maybe I'm mistaken.

**Comment by ludicast** [comment]
made pr with added support in https://github.com/HewlettPackard/dockerfile-parser-rs/pull/28. 

(so I can hopefully use in `dprint` btw :))

**Comment by dsherret** [comment]
Fixed in #28
